src/vt/configs/arguments/args.cc

```diff
@@ -186,6 +374,256 @@ std::tuple<int, std::string> parseArguments(
   return std::make_tuple(-1, std::string{});
 }
 
+void addYamlConfig(CLI::App& app, AppConfig& appConfig) {
```

Comment:
Rename this for consistency:
```suggestion
void addYamlArgs(CLI::App& app, AppConfig& appConfig) {
```